pkg/client/cache/fifo.go

```diff
@@ -241,6 +241,12 @@ func (f *FIFO) Replace(list []interface{}, resourceVersion string) error {
 	return nil
 }
 
+// Resync will touch all objects to put them into the processing queue
+func (f *FIFO) Resync() error {
+	// Nothing to do
+	return nil
```

Comment:
A concern I had doing this was adding an item into the processing queue twice.  If something is in f.items and f.queue and then resync is called, the result would be some items in f.queue twice unless we try to verify whether the item exists already.

The bigger question in my mind is whether iterating over f.items and placing into f.queue made sense.  Since this is a FIFO one would expect the items to be removed when processed, but that isn't a guarantee.

Do we care if we add an item into the queue twice?